app/views/videos/index.html.haml

```diff
@@ -1,9 +1,9 @@
 - @categories.each do |category|
-  %article.video_category
+  %article.video_category(id="#{category.name.downcase}")
     %header
       %h3= link_to category.name, category_path(category)
     .videos.row
       - category.recent_videos.each do |video|
         .video.col-sm-2
-          = link_to video_path(video)  do
-            %img(src="#{video.small_cover_url}")
+          = link_to video_path(video) do
+            %img(src="#{video.small_cover_url}" alt="#{video.title}")
```

Comment:
I would recommend to use `= image_tag ...`. Its more succinct and easier to read.